kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -290,6 +290,28 @@ paths:
                 items:
                   $ref: '#/components/schemas/ConsumerGroup'
 
+  /api/clusters/offsets/{clusterName}:
```

Comment:
Why do we need special endpoint? Could we add this to exist topic partition list?